package.json

```diff
@@ -70,5 +43,9 @@
     "semver": "^5.1.0",
     "source-map": "^0.5.1",
     "split": "^1.0.0"
+  },
+  "scripts": {
+    "test": "./bin/run-test.sh",
```

Comment:
We should preserve all scripts for now then so that we don't have dead code/deps.